packages/calypso-e2e/src/data-helper.ts

```diff
@@ -172,7 +190,7 @@ export function getAccountCredential( accountType: TestAccountName ): AccountCre
  */
 export function getAccountSiteURL(
 	accountType: TestAccountName,
-	{ protocol = true }: { protocol?: boolean } = {}
+	{ protocol = false }: { protocol?: boolean } = {}
```

Comment:
**Fix (blocking)**: As a heads up, it looks like this change in the default might break the P2 spec here:
https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/p2/p2__post.ts#L32

... Which relies on the protocol being added by default. I think it's fine either way the default goes, just wanted to avoid the breaking change!

Reply:
Good catch, thanks. I'll take a look.

Reply:
Change removed from the PR.